perllib/FixMyStreet/Cobrand/Buckinghamshire.pm

```diff
@@ -410,6 +411,60 @@ sub lookup_site_code_config { {
 around 'munge_sendreport_params' => sub {
     my ($orig, $self, $row, $h, $params) = @_;
 
+    if ($row->category eq 'Claim') {
+        my @photos = grep { $_ } (
+            $row->photo,
+            $row->get_extra_metadata('vehicle_photos'),
+            $row->get_extra_metadata('property_photos'),
+        );
+        my $photoset = FixMyStreet::App::Model::PhotoSet->new({
+            db_data => join(',', @photos),
+        });
+
+        my $num = $photoset->num_images;
+        my $id = $row->id;
+        my @attachments;
+        foreach (0..$num-1) {
+            my $image = $photoset->get_raw_image($_);
+            push @attachments, {
+                body => $image->{data},
+                attributes => {
+                    filename => "$id.$_." . $image->{extension},
+                    content_type => $image->{content_type},
+                    encoding => 'base64', # quoted-printable ends up with newlines corrupting binary data
+                    name => "$id.$_." . $image->{extension},
+                },
+            };
+        }
+
+        my @files = grep { $_ } (
+            $row->get_extra_metadata('v5'),
+            $row->get_extra_metadata('vehicle_receipts'),
+            $row->get_extra_metadata('tyre_receipts'),
+            $row->get_extra_metadata('property_insurance'),
+            $row->get_extra_metadata('property_invoices'),
+        );
+        foreach (@files) {
+            my $filename = $_->{filenames}[0];
+            my $id = $_->{files};
+            my $dir = FixMyStreet->config('PHOTO_STORAGE_OPTIONS')->{UPLOAD_DIR};
+            $dir = path($dir, "claims_files")->absolute(FixMyStreet->path_to());
+            my $data = path($dir, $id)->slurp_raw;
+            push @attachments, {
+                body => $data,
+                attributes => {
+                    filename => $filename,
+                    #content_type => $image->{content_type},
+                    encoding => 'base64', # quoted-printable ends up with newlines corrupting binary data
+                    name => $filename,
+                },
+            };
+        }
+
+        $params->{_attachments_} = \@attachments;
```

Comment:
I did manage to get an "email too large error" which still registered the report as sent by attaching two 5 Mb PDFs.